INN/Largo
# Adding aria-current detection to widget-content as used by recent-pos…

…ts and series posts

## Changes

This pull request makes the following changes:

- ```widget-content.php``` was updated to test for display of the current page and if that test is positive add the ```aria-current="page"``` attribute to the anchor tag. Any widget that uses this partial will have this test run and apply the attribute as appropriate.

## Why

This PR suggests changes that allows Largo to match the future behavior of WordPress' widgets
For #1805 

## Testing/Questions


Questions that need to be answered before merging:


Steps to test this PR:

1. I added the recent posts widget to my local Largo installation
2. Navigate to a piece of content in the recent posts list
3. View source / inspect the anchor tag link of the recent post that is the same post as the one being viewed
4. Check for the existence of the aria-current="page" attribute in the anchor tag
5. Navigate to another piece of content
6. Repeat step 3 and check that the attribute is not added to the anchor tag

## Additional information

INN Member/Labs Client requesting: (if applicable)
